tests/conftest.py

```diff
@@ -1570,6 +1571,46 @@ def finalizer():
     return factory
 
 
+@pytest.fixture(scope="session", autouse=True)
+def obc_deployment_factory_fixture(request):
+    """
+    Creates deployment
+    Calling this fixture creates a new deployment for obc
+    """
+    instances = []
+
+    def factory(
+        namespace=constants.OPENSHIFT_STORAGE_NAMESPACE,
+        replicas=3,
+        number_of_deployments=1,
+    ):
+        """
+        name (str): name of the deployment for obc
+        namespace (str): namespace where the deployment should be running
+        replicas (int) : replicas of the deployment
+
+        """
+        if number_of_deployments > 1:
+            deploy_objs = helpers.create_multiple_deployments(
+                number_of_deployments=number_of_deployments
+            )
+        else:
+            deploy_objs = helpers.create_deployment_for_obc(replicas=replicas)
+
+        instances.append(deploy_objs)
```

Comment:
Should we set the instances variable equal to an empty list to make it obvious that it's iterable?

Reply:
yes